backend/api/views.py

```diff
@@ -1,3 +1,19 @@
-from django.shortcuts import render
+from rest_framework import viewsets, mixins
+from serializers import BusinessSerializer
+
+from sodapy import Socrata
+
+class BusinessesViewSet(mixins.ListModelMixin,
+                        viewsets.GenericViewSet):
+
+    serializer_class = BusinessSerializer
+
+    def get_queryset(self):
+        client = Socrata("data.cityofnewyork.us", "z9bHXnSMLOVBFTGrELvWCcCBN")
+        businesses = client.get("9w7m-hzhe", content_type="json",
+                                limit="500000",
+                                select="camis, dba, boro, building, street, zipcode, cuisine_description")
```

Comment:
Rubber ducking on myself: 
Since i can't use `group by` as a query parameter here without errors once a business instance is missing a key-value pair, i was thinking another thing i can do is split this into multiple calls by adding an `order="camis"` and an offset. And then once i have responses of chunks of business data, i can get all the unique ones after like i did below. Would this be faster? 

I guess this is the bigger question:
http://softwareengineering.stackexchange.com/questions/274340/many-small-requests-vs-few-large-requests-api-design